d2game/d2player/globeWidget.go

```diff
@@ -95,28 +88,43 @@ func newGlobeWidget(ui *d2ui.UIManager, x, y int, gtype globeType, value, valueM
 		}
 	}
 
-	return &globeWidget{
+	gw := &globeWidget{
 		BaseWidget: base,
 		value:      value,
 		valueMax:   valueMax,
 		globe:      globe,
 		overlap:    overlap,
 	}
+
+	gw.logger = d2util.NewLogger()
+	gw.logger.SetLevel(l)
+	gw.logger.SetPrefix(logPrefix)
+
```

Comment:
I think that this is problematic. Do individual widgets need logger instances?

should we not instead return these errors to the HUD and let the HUD report errors?

Reply:
yes, we can return errors to HUD, but ONLY ERRORS. What about logger.Info like in help_overlay:go:202?